part1/anecdote/src/App.jsx

```diff
@@ -0,0 +1,59 @@
+import { useState } from 'react'
+
+const Button = ({onClick, text}) => <button onClick={onClick}>{text}</button>
+
+const Display = ({anecdotes, points}) => {
+  return (
+    <div>
+      <p>{anecdotes}</p>
+      <p>has {points} points</p>
+    </div>
+  )
+}
+
+const App = () => {
+
+  function getRandomInt(max) {
+    return Math.floor(Math.random() * max);
+  }
+
+  const handleNextAnecdote = () => {
+    return setSelected(getRandomInt(8))
+  }
+
+  const handleVote = () => {
+    const copy = [...points]
+    copy[selected] += 1
+    setPoints(copy)
```

Comment:
```suggestion
    setPoints((p) => p[selected] += 1)
```
I think this works when you use the setter, but maybe you do need the copy of the array to stop it from shitting its pants